media/js/firefox/new/better-browser-experiment.js

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    // courtesy of https://davidwalsh.name/query-string-javascript
+    function getUrlParam(name) {
```

Comment:
Could using the existing `search-params.js` lib be an alternative to re-implementing this here? https://github.com/mozilla/bedrock/blob/master/media/js/base/search-params.js

Reply:
That was my first thought/attempt, but `search-params.js` depends on jQuery, and since this is a pre-render script...

Reply:
Bah jQuery :/